src/CoreBundle/Service/TreeService.php

```diff
@@ -74,6 +74,10 @@ public function getChildren($treeId, $includeHidden = false, $languageId = null)
             }
         }
 
+        /**
+         * @psalm-suppress InvalidArgument
+         * @FIXME returning `null` from a sorting method is not allowed, should probably return `0`.
+         */
         usort($children, function (TdbCmsTree $a, TdbCmsTree $b) {
```

Comment:
This sorting-closure returns `null` in an 'undecided' case - should probably be `0`